tests/perf/run_perf_tests.py

```diff
@@ -29,16 +29,16 @@
 def usage(argv):
     """ Print script usage
     """
-    print("Usage: " + argv[0] + " -h -p vegetaPatternTarFile -c daemonOnCore  -t turboGethAddress -g turboGethHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence")
+    print("Usage: " + argv[0] + " -h -p vegetaPatternTarFile -c daemonOnCore  -t turboErigonAddress -g turboErigonHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence")
     print("")
     print("Launch an automated performance test sequence on Silkrpc and RPCDaemon using Vegeta")
     print("")
     print("-h                      print this help")
     print("-d rpcDaemonAddress     address of daemon eg (10.1.1.20)                                                       [default: " + DEFAULT_RPCDAEMON_ADDRESS +"]")
     print("-p vegetaPatternTarFile path to the request file for Vegeta attack                                             [default: " + DEFAULT_VEGETA_PATTERN_TAR_FILE +"]")
     print("-c daemonVegetaOnCore   cpu list in taskset format for daemon & vegeta (e.g. 0-1:2-3 or 0-2:3-4 or 0,2:3,4...) [default: " + DEFAULT_DAEMON_VEGETA_ON_CORE +"]")
-    print("-a turboGethAddress     address of TG Core component as <address>:<port> (e.g. localhost:9090)                 [default: " + DEFAULT_TG_ADDRESS + "]")
-    print("-g turboGethHomeDir     path to TG home folder (e.g. ../../../turbo-geth/)                                     [default: " + DEFAULT_GETH_BUILD_DIR + "]")
+    print("-a erigonAddress        address of ERIGON Core component as <address>:<port> (e.g. localhost:9090)             [default: " + DEFAULT_ERIGON_ADDRESS + "]")
+    print("-g erigonHomeDir        path to ERIGON home folder (e.g. ../../../erigon/)                                     [default: " + DEFAULT_ERIGON_BUILD_DIR + "]")
```

Comment:
Same comment as above for `erigonHomeDir`

Reply:
ok